assets/scripts/Map/Bridge/blockPlatform.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+const { ccclass, property } = cc._decorator;
+
+@ccclass
+export default class blockPlatform extends cc.Component {
+  private aux = 0;
+
+  private isInContact = false;
+
+  public onBeginContact(_contact: Event, _selfCollider: cc.BoxCollider, _otherCollider: cc.BoxCollider): void {
```

Comment:
Essas variáveis não estão sendo usadas, e não são obrigatórias, coloque apenas as que vai usar, e caso precise de uma mais a frente, insira um _nomeDaVariavel no lugar do nome, para mostrar que ela é inútil.